bit/has_alternative_bit.py

```diff
@@ -0,0 +1,31 @@
+"""
+Given a positive integer, check whether it has alternating bits: namely,
+if two adjacent bits will always have different values.
+
+For example:
+Input: 5
+Output: True because the binary representation of 5 is: 101.
+
+Input: 7
+Output: False because the binary representation of 7 is: 111.
+
+Input: 11
+Output: False because the binary representation of 11 is: 1011.
+
+Input: 10
+Output: True because The binary representation of 10 is: 1010.
+"""
+
+def has_alternative_bit(n):
```

Comment:
This can also be done using bit mask in O(1). Should I commit that algo in this same PR?

Reply:
I merge your commit in my RP. Thanks